### PR TITLE
Feature/related devices

### PR DESCRIPTION
**Pull request by hassan-yonomi**

### CODE REVIEW

File: lib/device.js [hunk 1]
```diff
@@ -45,9 +45,17 @@ class Device extends DeviceSDK {
       data => {
         log.info({ data }, 'device connected');
 
+        // this.eventSource.on('relatedDevices/commands', cmd => {
```
Comment (varddum) [inline lib/device.js]: Should we delete this?

File: lib/device.js [hunk 2]
```diff
@@ -90,12 +98,12 @@ class Device extends DeviceSDK {
             });
           }
 
-          if (cmd.method === '_update') {
-            if (cmd.params[0].data.custom) {
-              this.state = cmd.params[0].data.custom;
-              this.update({ custom: this.state });
-            }
-          }
+          // if (cmd.method === '_update') {
```
Comment (varddum) [inline lib/device.js]: Or this?

**Review by yonomi-developer — APPROVED** [review]

File: lib/device.js [hunk 3]
```diff
@@ -45,9 +45,12 @@ class Device extends DeviceSDK {
...
         log.info({ data }, 'device connected');
 
-        // Add commands and code to implment commands
         this.eventSource.on('command', cmd => {
 
+          if (cmd.method === 'noop') {
+
+          }
+
           if (cmd.method === 'ping') {
```
[the remark this answers is not in the document]
Comment (hassan-yonomi) [reply lib/device.js]: we dont really have an else case like it dosent have to be in chain to evaluate conditions we are just checking the method name and doing some business logic. There is no else case in here either we handle a command based on its method name or not.

[the remark this answers is not in the document]
Comment (stewartjarod) [reply lib/device.js]: They should be but also shouldn't matter

**Review by stewartjarod — APPROVED** [review]

**Pull request merged**